DBconnect/DBconnection.cs

```diff
@@ -258,7 +258,7 @@ public void Close()
                 error?.Invoke("Соединенине с БД либо уже закрыто, либо его не было");
```

Comment:
```suggestion
                Error?.Invoke("Соединенине с БД либо уже закрыто, либо его не было");
```
Другое соглашение об именовании переменных в этом проекте было? Если нет, то имена эвентов и делегатов пишутся с большой буквы